app/electron/main.ts

```diff
@@ -671,12 +682,15 @@ function startElecron() {
      * 'command-stderr', and 'command-exit' events back to the renderer
      * process with the command's output and exit code.
      *
+     * In case of 'electron-node' command, it uses fork instead of spawn
```

Comment:
Have you tried this on different platforms? 
(I haven't tried it with Electron, but forking a GUI process with threads xplatform is a tricky thing in my experience.)

Reply:
Will get it checked, works perfectly in WSL Ubuntu on x86_64 machine.